upgrade-tests/src/test/java/io/zeebe/test/UpgradeTest.java

```diff
@@ -100,6 +103,42 @@
                 .afterUpgrade(UpgradeTest::publishMessage)
                 .done()
           },
+          // TODO (saig0): enable the test case when upgrading from 0.25.0
```

Comment:
Can remove this, right? It is not relevant for 0.24.

Reply:
True. I can remove it. It will never work for this version :sweat_smile: